src/components/CustomNavLink.tsx

```diff
@@ -6,11 +6,13 @@ const CustomNavLink = ({ path, faIcon, name }: CustomNavLinkProps) => {
   return (
     <NavLink
       className={({ isActive }) =>
-        isActive ? 'nav-link active-link' : 'nav-link'
+        isActive
+          ? 'flex px-6 font-bold text-[#9dbaff]  rounded-md'
+          : 'flex px-6 font-bold text-white hover:text-[#9dbaff] rounded-md'
       }
       to={path}
     >
-      <div className="icon-wrapper">
+      <div className="flex justify-center items-center mr-4">
```

Comment:
tailwind css is very nice and fast, but the downside of using utility-first-class based framework like tailwind is that it becomes hard to read ( you lose semantics), in other words. you don't know what is the purpose of html elements, and since you will be working with other programmers it is best to write code that is understandable (ideally from the first glance). 

 you have 2 solutions here. you can introduce semantics via built-in html elements. you can start from  [there](https://web.dev/learn/html/semantic-html/)  - try to avoid div.  and secondly, you can introduce data-* attributes like so: 
`<div className="flex justify-center items-center mr-4" data-purpose="icon-wrapper">`